data/sql/updates/pending_db_world/rev_1696800156876281900.sql

```diff
@@ -0,0 +1,570 @@
+(@OGUID+10, @OBJECTPOOLS+1, 0, 'Durotar Earthroot NW 7/14'),
+(@OGUID+11, @OBJECTPOOLS+1, 0, 'Durotar Earthroot NW 8/14'),
+(@OGUID+12, @OBJECTPOOLS+1, 0, 'Durotar Earthroot NW 9/14'),
+(@OGUID+13, @OBJECTPOOLS+1, 0, 'Durotar Earthroot NW 10/14'),
+(@OGUID+14, @OBJECTPOOLS+1, 0, 'Durotar Earthroot NW 11/14'),
+(@OGUID+15, @OBJECTPOOLS+1, 0, 'Durotar Earthroot NW 12/14'),
+(@OGUID+16, @OBJECTPOOLS+1, 0, 'Durotar Earthroot NW 13/14'),
+(@OGUID+17, @OBJECTPOOLS+1, 0, 'Durotar Earthroot NW 14/14');
+
+INSERT INTO `pool_template` (`entry`, `max_limit`, `description`) VALUES
+(@OBJECTPOOLS+2, 2, 'Durotar Earthroot NE Pool 3/5');
+INSERT INTO `pool_gameobject` (`guid`, `pool_entry`, `chance`, `description`) VALUES
+(@OGUID+19, @OBJECTPOOLS+2, 0, 'Durotar Earthroot NE 1/5'),
+(@OGUID+20, @OBJECTPOOLS+2, 0, 'Durotar Earthroot NE 2/5'),
+(@OGUID+21, @OBJECTPOOLS+2, 0, 'Durotar Earthroot NE 3/5'),
+(@OGUID+22, @OBJECTPOOLS+2, 0, 'Durotar Earthroot NE 4/5'),
+(@OGUID+23, @OBJECTPOOLS+2, 0, 'Durotar Earthroot NE 5/5');
+
+INSERT INTO `pool_template` (`entry`, `max_limit`, `description`) VALUES
+(@OBJECTPOOLS+3, 6, 'Durotar Earthroot Center Pool 4/5');
+INSERT INTO `pool_gameobject` (`guid`, `pool_entry`, `chance`, `description`) VALUES
+(@OGUID+24, @OBJECTPOOLS+3, 0, 'Durotar Earthroot Center 1/17'),
+(@OGUID+25, @OBJECTPOOLS+3, 0, 'Durotar Earthroot Center 2/17'),
+(@OGUID+26, @OBJECTPOOLS+3, 0, 'Durotar Earthroot Center 3/17'),
+(@OGUID+27, @OBJECTPOOLS+3, 0, 'Durotar Earthroot Center 4/17'),
+(@OGUID+28, @OBJECTPOOLS+3, 0, 'Durotar Earthroot Center 5/17'),
+(@OGUID+29, @OBJECTPOOLS+3, 0, 'Durotar Earthroot Center 6/17'),
+(@OGUID+30, @OBJECTPOOLS+3, 0, 'Durotar Earthroot Center 7/17'),
+(@OGUID+31, @OBJECTPOOLS+3, 0, 'Durotar Earthroot Center 8/17'),
+(@OGUID+32, @OBJECTPOOLS+3, 0, 'Durotar Earthroot Center 9/17'),
+(@OGUID+33, @OBJECTPOOLS+3, 0, 'Durotar Earthroot Center 10/17'),
+(@OGUID+34, @OBJECTPOOLS+3, 0, 'Durotar Earthroot Center 11/17'),
+(@OGUID+35, @OBJECTPOOLS+3, 0, 'Durotar Earthroot Center 12/17'),
+(@OGUID+36, @OBJECTPOOLS+3, 0, 'Durotar Earthroot Center 13/17'),
+(@OGUID+37, @OBJECTPOOLS+3, 0, 'Durotar Earthroot Center 14/17'),
+(@OGUID+38, @OBJECTPOOLS+3, 0, 'Durotar Earthroot Center 15/17'),
+(@OGUID+39, @OBJECTPOOLS+3, 0, 'Durotar Earthroot Center 16/17'),
+(@OGUID+40, @OBJECTPOOLS+3, 0, 'Durotar Earthroot Center 17/17');
+
+INSERT INTO `pool_template` (`entry`, `max_limit`, `description`) VALUES
+(@OBJECTPOOLS+4, 2, 'Durotar Earthroot South Pool 5/5');
+INSERT INTO `pool_gameobject` (`guid`, `pool_entry`, `chance`, `description`) VALUES
+(@OGUID+41, @OBJECTPOOLS+4, 0, 'Durotar Earthroot South 1/5'),
+(@OGUID+42, @OBJECTPOOLS+4, 0, 'Durotar Earthroot South 2/5'),
+(@OGUID+43, @OBJECTPOOLS+4, 0, 'Durotar Earthroot South 3/5'),
+(@OGUID+44, @OBJECTPOOLS+4, 0, 'Durotar Earthroot South 4/5'),
+(@OGUID+45, @OBJECTPOOLS+4, 0, 'Durotar Earthroot South 5/5');
+
+--TODO:  CHECK ALL THESE AND THEN POOL THEM:
```

Comment:
```suggestion
-- TODO:  CHECK ALL THESE AND THEN POOL THEM:
```

May need to add a space here for workflow.

Reply:
that needed removed entirely actually, it was my note to self from last night